util/chplenv/utils.py

```diff
@@ -62,14 +62,22 @@ def try_run_command(command, stdout=True, stderr=False, cmd_input=None):
 def run_command(command, stdout=True, stderr=False, cmd_input=None):
     """Command subprocess wrapper.
        This is the usual way to run a command and collect its output."""
-    exists, returncode, output = try_run_command(command, stdout, stderr,
-                                                 cmd_input)
+    exists, returncode, myOutput = try_run_command(command, True, True,
```

Comment:
nit: python uses snake_case, e.g.

```suggestion
    exists, returncode, my_output = try_run_command(command, True, True,
```